platform/sensor-stream-manager/sensor-stream-manager.js

```diff
@@ -1,64 +1,205 @@
 const fs = require("fs-extra");
 const mqtt = require("mqtt");
 const utils = require("../../utils");
+const fetch = require('node-fetch');
 
 // TODO: add mqtt-data-collector logic to SSM.
 const MessagingService = require('../messaging-service');
 
-console.log("[INFO] Initialize app-manager...");
-const serviceName = process.env.SERVICE_NAME;
-const messagingService = new MessagingService(serviceName);
+class SensorStreamManager {
```

Comment:
Switch to a non-class implementation. Investigate on improving readability for nodejs files.